doc/api_callback.rst

```diff
@@ -80,12 +80,16 @@
     `action` value     meaning                                                                           Appearance*
     ================== ================================================================================= ===========
     PKG_CLEANUP        `package` cleanup is being performed.                                             3
-    PKG_DOWNGRADE      `package` is being downgraded.                                                    2
+    PKG_DOWNGRADE      new `package` is being installed as a downgrade                                   2
+    PKG_DOWNGRADED     installed `package` is being downgraded.                                          2
     PKG_INSTALL        `package` is being installed.                                                     2
-    PKG_OBSOLETE       `package` is being obsoleted.                                                     2
-    PKG_REINSTALL      `package` is being reinstalled.                                                   2
+    PKG_OBSOLETE       new `package` is obsoleting a package.                                            2
+    PKG_OBSOLETED      installed `package` is being obsoleted.                                           2
+    PKG_REINSTALL      new `package` is installed as reinstall.                                          2
```

Comment:
> ... as **a** reinstall.

Similar to the above (and a couple more lines), it doesn't seem to make sense (_new package as a reinstall_ even seems to be an oxymoron :slightly_smiling_face: )